pkg/foundation/metrics/measure/measure.go

```diff
@@ -41,6 +41,12 @@ var (
 		// buckets from 1KiB to 2MiB
 		prometheus.HistogramOpts{Buckets: []float64{1024, 1024 << 1, 1024 << 2, 1024 << 3, 1024 << 4, 1024 << 5, 1024 << 6, 1024 << 7, 1024 << 8, 1024 << 9, 1024 << 10, 1024 << 11}},
 	)
+	DLQBytesHistogram = metrics.NewLabeledHistogram("conduit_dlq_bytes",
+		"Number of bytes a DLQ connector processed per pipeline and plugin.",
```

Comment:
Nitpick: it would be nice to have a few words about what these bytes are (similar to what you already did in RecordBytesHistogram).

Reply:
Do you mean the help text? I wouldn't add too much info there, looking at existing metrics these help texts are kept very brief. If someone is interested in the details they can have a look at the docs.

That said, I agree that we should mention how we calculate the record bytes in our metrics docs, I'll do that in a separate PR!

Reply:
https://github.com/ConduitIO/conduit/pull/829